.github/workflows/ci.yml

```
@@ -17,7 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.0.0
```

```suggestion
    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4
```

If you use v4 then it'll automatically pick up the latest, i.e. if we release 4.0.1 then it'll use that.

---

Thanks, will do that

But then I'm not understanding something

Why didn't this work with `rate-limit`? Using v4 there with the redis workflow resulted in errors

https://github.com/fastify/fastify-rate-limit/pull/350
https://github.com/fastify/fastify-rate-limit/actions/runs/7195862716

---

There you go, it didn't work...

I think it might be because `v4` doesn't exist as a release/tag

---

Ah yeah, forgot the v4 release went awry: https://github.com/fastify/workflows/pull/107, grumble grumble grumble.